samples/client/petstore/java/jersey2/src/main/java/io/swagger/client/ApiClient.java

```diff
@@ -575,7 +576,7 @@ public String escapeString(String str) {
   public File downloadFileFromResponse(Response response) throws ApiException {
     try {
       File file = prepareDownloadFile(response);
-      Files.copy(response.readEntity(InputStream.class), file.toPath());
+	  Files.copy(response.readEntity(InputStream.class), file.toPath(), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
@HaraldWalker The sample is using tab instead of 4-space. Please regenerate the sample when you've time.